doc/.static/version_switch.js

```diff
@@ -10,9 +9,9 @@
   // If doc is run localy
   if (url.startsWith('file')) {
     theano_dir = 'html';
-    versions_dir = {"local":"html"};
+    versions_dir = {"local":"html", "test":"test"};
```

Comment:
It only appears when the doc is run locally (url startsWith file). I use it
to make sure the version list is formatted properly when testing locally.

On Wed, Jun 22, 2016 at 9:50 PM, Pascal Lamblin <notifications@github.com>
wrote:

> In doc/.static/version_switch.js
> <https://github.com/Theano/Theano/pull/4604#discussion_r68165176>:
>
> > @@ -10,9 +9,9 @@
> >    // If doc is run localy
> >    if (url.startsWith('file')) {
> >      theano_dir = 'html';
> > -    versions_dir = {"local":"html"};
> > +    versions_dir = {"local":"html", "test":"test"};
>
> Should "test" here stay in the final version?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4604/files/b6a7bd28fe014655b358a66f20ae535821dfb439#r68165176>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ARWYKZ_RB2TP2DgXOUZSIIUuZ9aatbq6ks5qOeaAgaJpZM4IxCFs>
> .
>